index.html

```diff
@@ -0,0 +1,177 @@
+        <h3>PEGS Exihibition</h3>
+        <p class="text-light">Appreciate various creations applying PEGS license of physical therapists, organized from
+          wellness centers accross Nairobi.
+        </p>
+      </li>
+      <li class="list">
+        <i class="material-icons">forum</i>
+        <h3>Forum</h3>
+        <p class="text-light">Try creating your own routine using open source license rather than just watching.</p>
+      </li>
+      <li class="list">
+        <i class="material-icons">home_repair_service</i>
+        <h3>Workshop</h3>
+        <p class="text-light">Try creating your own work using open source license rather than just watching.</p>
+      </li>
+      <li class="list">
+        <i class="material-icons">attractions</i>
+        <h3>PEGS Ignite</h3>
+        <p class="text-light">Get the opportunities to network with PEGS affliates around the world, also after the
+          summit.</p>
+      </li>
+    </ul>
+    <button type="button">Join PEGS Global Summit 2023</button>
+    <article class="see-program text-light" id="see-program"><i class="material-icons">arrow_right_alt</i> click here to
+      .<a href="#speakers" class="text-light"><b><u> SEE THE FEATURED SPEAKERS</u></b></a>
+    </article>
+  </section>
+
+  <!-- Featured Speakers -->
+  <section class="speakers hidden" id="speakers">
+  </section>
+
+  <section class="partners hidden" id="partners">
+    <div>
+      <h2 class="text-light">Partners</h2>
+      <div class="orange-line"></div>
+    </div>
+    <ul>
+      <li>Fitness & Gym Kenya</li>
+      <li>Google</li>
+      <li>Alpha Fit</li>
+      <li>Addidas</li>
+      <li>Evib Sports Science - Gym</li>
+    </ul>
+  </section>
+  <footer class="footer hidden" id="footer">
+    <div>
+      <i class="material-icons">school</i>
+      <h2>Afya Bora Nutrition and Wellness Center - Nairobi</h2>
+    </div>
+    <div>
+      Afya Bora Nutrition and Wellness Center - Nairobi
+      <br>
+      All rights reserved
+    </div>
+  </footer>
+  <div class="see-more" id="see-more">
+    <div>See more</div>
+    <i class="material-icons">keyboard_arrow_down</i>
+  </div>
+  <div class="see-less hidden" id="see-less">
+    <div>See less</div>
+    <i class="material-icons">keyboard_arrow_up</i>
+  </div>
+  <!-- Mobile menu -->
+  <section class="mobile-menu hidden" id="mobile-menu">
+    <ul class="m-menu">
+      <a href="about.html" class="text-light">
+        <li>About</li>
+      </a>
+      <a href="#main-program" class="text-light">
+        <li>Program</li>
+      </a>
+      <a href="#" class="text-light">
+        <li>Join</li>
+      </a>
+      <a href="#" class="text-light">
+        <li>Sponsor</li>
+      </a>
+      <a href="#" class="text-light">
+        <li>News</li>
+      </a>
+      <a href="#" class="text-light">
+        <li>PEGS Campaign</li>
+      </a>
+    </ul>
+    <span class="glyphicon glyphicon-remove" id="glyphicon-remove"></span>
+  </section>
```

Comment:
- Hey, to enforce the correct use of the html5 semantics and layout, the footer is supposed to be the last section on your website, so no other section should come after it. You could kindly move these sections up to the header to make your codebase more readable and organized.